css/style.css

```diff
@@ -8,656 +8,1472 @@
+    font-size: 40px;
+    line-height: 50px;
+    letter-spacing: 0.37px;
+    color: #172b4d;
+    margin-top: 70%;
+    margin-left: -70%;
+    align-items: flex-start;
+  }
+
+  .myself-text {
+    color: #344563;
+    font-size: 16px;
+    font-weight: 400;
+    font-family: "Poppins", sans-serif;
+    margin-left: 25px;
+  }
+
+  .about-submit-btn {
+    border-radius: 10px;
+    background-color: #fff;
+    padding: 10px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: flex;
+    cursor: pointer;
+    border: 1px solid #396df2;
+    margin-left: -65px;
+  }
+
+  .about-submit-btn a {
+    text-decoration: none;
+    font-size: 25px;
+    line-height: 20px;
+    color: #396df2;
+    font-weight: 500;
+    padding: 10px;
+    align-items: center;
+    font-style: bold;
+  }
+
+  .social-about {
+    align-self: flex-start;
+    margin-bottom: 5%;
+  }
+
+  .connection-about {
+    height: 24px;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+    font-weight: 900;
+    padding: 20px 0;
+    margin-left: 20px;
+  }
+
+  .media-icons-about {
+    display: flex;
+    align-content: center;
+    gap: 15px;
+    left: 6.4%;
+    bottom: 18.88%;
+    top: 77.81%;
+    width: 100%;
+    padding-top: 20px;
+    color: #505f79;
+    margin-left: 20px;
+  }
+
+  .lang {
+    display: grid;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+    width: 365px;
+    margin-left: 5%;
+    margin-top: 10px;
+  }
+
+  .about-lang {
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 50px;
+    letter-spacing: 0.37px;
+    margin-top: 20%;
+    margin-left: -1%;
+    font-family: "Poppins", sans-serif;
+    align-items: flex-start;
+  }
+
+  .lang-list {
+    display: flex;
+    margin-left: -4%;
+    height: 300px;
+    padding: 10px;
+    background: #f7f7f9;
+    flex-direction: column;
+  }
+
+  .works-frame {
+    margin-top: -70px;
+    width: 100%;
+    height: 250px;
+    margin-left: -4%;
+    padding: 10px;
+  }
+
+  .display-skills {
+    border: 1px solid black;
+    list-style: none;
+  }
+
+  .frame-work {
+    list-style: none;
+    display: flex;
+    gap: 25px;
+    color: #000;
+    order: 1;
+    margin: 35px;
+    margin-top: 70px;
+    align-items: center;
+    margin-left: -2%;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+  }
+
+  .left-arrow {
+    display: flex;
+    margin-left: 55%;
+    left: 41.67%;
+    right: 33.33%;
+    top: 25%;
+    bottom: 2;
+  }
+
+  .left-arrow2 {
+    display: flex;
+    margin-left: 80%;
+    left: 41.67%;
+    right: 33.33%;
+    top: 25%;
+    bottom: 2;
+  }
+
+  .seperator-btn {
+    width: 327px;
+    height: 1px;
+    left: 0;
+    right: 0;
+    padding: 10px;
+    bottom: 0;
+    background: #dfe1e6;
+  }
+
+  .contanct-containter {
+    display: flex;
+    margin-top: 60%;
+    max-width: 365px;
+    margin-left: -5%;
+    background-color: #6070ff;
+    border-top-left-radius: 100px;
+  }
```

Comment:
- Kindly, be sure your mobile version design still sticks to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=xmWXLXW7NiSVxIum-0), you lose your mobile style when you style the desktop. 👍👌


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206548470-985b609a-a380-452f-8a6c-26578dff7c6d.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206548484-f85eec4a-885a-48cb-8353-ad12d3ae4b52.png" width="200">|


😎️